src/server/lib/middleware/cache.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response, NextFunction } from 'express';
+
+// middleware that adds headers to disable caching on certain particular routes,
```

Comment:
That makes sense, I can have noCache enabled by default, and have another method to reset any routes that we'd want to override

Reply:
- Updated so that `noCache` is applied by default on the reset and change password routes (https://github.com/guardian/gateway/pull/13/files#diff-c5708ff75525759b3738477fa6c8198aR14-R18)

- Added the `removeNoCache` middleware which removes the no-cache headers

- I've not added a `cache` middleware as we currently don't have any routes that need explicit caching

Reply:
🥳